### PR TITLE
upgrade avaloniaui and some nuget else

### PR DESCRIPTION
**Pull request by NeverMorewd**

### CODE REVIEW

File: src/ScrcpyNet.Benchmark/Benchmark.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 namespace ScrcpyNet.Benchmark
 {
-    [SimpleJob(RuntimeMoniker.CoreRt31)]
+    //[SimpleJob(RuntimeMoniker.CoreRt31)]
```
Comment (Peter-Schenkels) [inline src/ScrcpyNet.Benchmark/Benchmark.cs]: Why did you comment this out? This happens more than once, please either delete or add a comment why this should be left commented out.

File: src/ScrcpyNet/VideoStreamDecoder.cs [hunk 2]
```diff
@@ -26,7 +26,7 @@ public ReadOnlySpan<byte> Data
 
         public int Width { get; }
         public int Height { get; }
-        public int FrameNumber { get; }
+        //public int FrameNumber { get; }
```
Comment (Peter-Schenkels) [inline src/ScrcpyNet/VideoStreamDecoder.cs]: Please either delete or provide a reason why this code is left commented.

File: src/ScrcpyNet.Benchmark/Benchmark.cs (hunk 1 above)
Comment (NeverMorewd) [reply src/ScrcpyNet.Benchmark/Benchmark.cs]: I have upgraded benchmark.net to lastest version. RuntimeMoniker.CoreRt31 was renamed with RuntimeMoniker.NetCoreApp31. I will commit a new pull request to correct it.

File: src/ScrcpyNet/VideoStreamDecoder.cs (hunk 2 above)
Comment (NeverMorewd) [reply src/ScrcpyNet/VideoStreamDecoder.cs]: My fault. After upgrading FFmpeg.AutoGen, the AVCodecContext.frame_number(int) was refactored with AVCodecContext.frame_num(long)

**Comment by NeverMorewd** [comment]
I will close this pr, and then commit a new one.

**Pull request closed without merging**